code/modules/religion/sects/shadow_sect.dm

```diff
@@ -394,18 +375,36 @@
 	lisk.AddComponent(/datum/component/dark_favor, user)
 	lisk.set_light(sect.light_reach, sect.light_power, DARKNESS_INVERSE_COLOR)
 	playsound(altar_turf, 'sound/magic/fireball.ogg', 50, TRUE)
-	priority_announce("May our lord, [GLOB.deity], have mercy on your soul as darkness reigns apon you all.", "Faith Alert", SSstation.announcer.get_rand_alert_sound())
-	addtimer(CALLBACK(lisk, TYPE_PROC_REF(/obj/structure/destructible/religion/shadow_obelisk, final_darkness_activate)), 100)
+	if(user.mind.is_murderbone())
+		priority_announce("May our lord, [GLOB.deity], have mercy on your soul as darkness reigns apon you all.", "Faith Alert", SSstation.announcer.get_rand_alert_sound())
+	else
+		priority_announce("May our lord, [GLOB.deity], bless all the shadows as darkness heals those who worship the night.", "Faith Alert", SSstation.announcer.get_rand_alert_sound())
+	addtimer(CALLBACK(lisk, TYPE_PROC_REF(/obj/structure/destructible/religion/shadow_obelisk, final_darkness_activate), user), 250)
 
-/obj/structure/destructible/religion/shadow_obelisk/proc/final_darkness_activate()
+/obj/structure/destructible/religion/shadow_obelisk/proc/final_darkness_activate(mob/living/user)
 	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
+	var/list/candidates = poll_ghost_candidates("Do you wish to be summoned as a Shadow Faithful?", ROLE_HOLY_SUMMONED, null, 10 SECONDS, POLL_IGNORE_HOLYUNDEAD)
 	for(var/obj/structure/destructible/religion/shadow_obelisk/obs in sect.obelisks)
 		START_PROCESSING(SSobj, obs)
+		if(!user.mind.is_murderbone())
+			playsound(obs, 'sound/magic/fireball.ogg', 50, TRUE)
+			return
 		var/obelisk_turf = get_turf(obs)
-		for(var/i in 1 to 3)
-			var/mob/living/simple_animal/hostile/faithless/faithful/faithful = new(obelisk_turf)
-			faithful.AddComponent(/datum/component/dark_favor, faithful)
-			faithful.set_light(2, -2, DARKNESS_INVERSE_COLOR)
+		if(prob(30))
+			for(var/i in sect.min_summon to sect.max_summon)
```

Comment:
Don't just commit this suggestion, you'll need to unindent the contents too :(
```suggestion
		if(!prob(30))
		    continue
		for(var/i in sect.min_summon to sect.max_summon)
```

Reply:
Note that `prob(70)` is better than `!prob(30)`. using `not` (`!`) should be avoided for clean code.

Reply:
![image](https://github.com/user-attachments/assets/d2bb3b46-ff45-436c-a186-7bf2ebf77064)
 Like this?

Reply:
Beautiful